src/support/pagelocker.cpp

```diff
@@ -30,7 +30,7 @@
 #endif
 
 LockedPageManager* LockedPageManager::_instance = NULL;
-boost::once_flag LockedPageManager::init_flag = BOOST_ONCE_INIT;
+std::once_flag LockedPageManager::init_flag;
```

Comment:
I'd rather have those on a separate PR.
don't want to be pedantic. But there's really a lot of good and isolated changes in this PR that should be merged via separate changes, IMHO.

Reply:
submitted #1745